content/articles/css-hacks-for-web-developers/index.md

```diff
@@ -0,0 +1,178 @@
+
+Let's have a look at the example snippet below:
+```css
+img{
+  width: 500px;
+  height: 500px;
+  object-fit: scale-down;
+ }
+```
+In the above snippet, the `object-fit attribute specifies how an `image should be scaled to fit within its container.
+
+4. ### CSS style for visited links
+These styles are very important, yet some developers decide to avoid them. This sometimes causes trouble to the end-users as far as the usability of the web app is concerned.
+
+The `a: visited` link controls all the links that the visitor has clicked on, the `a: link` controls all links that haven’t been clicked on yet. while the `a: active` link shows a link a moment it is clicked. 
+
+Styling these links properly is a great benefit to the users. It builds a user-friendly website as users can navigate through the site easily
+
+Different colors are used for different link as shown in the snippet below:
+```css
+/* color red represent unvisited link */
+a:link {
+  color: red;
+}
+/* color green represent visited link */
+a:visited {
+  color: green;
+}
+/*color blue represent selected link */
+a:active {
+  color: blue
+```
+5. ### How to style Initial Letter using CSS
+The initial letter is a CSS technique that selects the first letter of a paragraph and specifies the number of lines it holds. It's usually used by print media and information sites, such as news sites. It's a very important technique as it grabs the reader's attention making them read the first line and finally the entire paragraph.
+
+While this style may appear to have dated, you can still design it to appear modern and profit from the psychological effect it has on your clients.
+
+Let's have a look at the example snippet below:
+```html
+<html>
+  <body>
+    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+    <style>
+       p:first-letter{
+       display: grid;
+       margin: 7px;
+       color: black;
+       font-size: 80px;
+       float: left;
+      }
+    </style>
+  </body>
+</html>
+```
+6. ### How to Group styling in CSS
+Styling different `HTML elements one by one is very tiresome and time wastage. This problem faced by developers is easily solved by learning how to consolidate CSS style.
+
+To do this, you separate different elements with commas (`,`) and add the CSS style inside, then you will have added the style to all of them.
+
+Because you don't have to write the same line of code numerous times, you'll save time and reduce the size of your code. Amazing Stuff🚀
+
+Let's have a look at the example snippet below:
+```html
+<html>
+  <body>
+    <h4>Why do developer Group elements together when styling?</h4>
+    <p>Developers Group elements together to save on time </p>
+    <h5>What are advantages Of Consolidating during styling?</h5>
+    <style>
+      /* suppose you want to add a background color, text color, font and center the 3 elements in the middle of the screec */
+      h4,p,h5{
+        background-color: black;
```

Comment:
In cases where different elements within different classes are the target, this method is not convenient. Please consider using classes and ids for this.